charts/drax/templates/secret-loki-tenant-override-config.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: reverse-proxy-auth-config
+type: Opaque
+stringData:
+  authn.yaml: |
+    users:
+      - username: notifications
+        password: accelleran
```

Comment:
We should stop using default passwords and start generating them randomly when installing. Maybe `upgrade` should then also be used instead of `uninstall` followed by `install` again as otherwise the password changes every time...

This is a larger effort I guess outside of the scope of this PR as there are other dependencies already having the same issue...